### PR TITLE
feat!(cmd): add vulncheck

### PR DESCRIPTION
**Pull request by MaineK00n**
If this Pull Request is work in progress, Add a prefix of “[WIP]” in the title.

# What did you implement:

add vulncheck-kev

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

```console
$ go-kev fetch vulncheck ~/vulncheck-kev
INFO[08-19|19:47:30] Fetching VulnCheck Known Exploited Vulnerabilities 
INFO[08-19|19:47:31] Insert VulnCheck Known Exploited Vulnerabilities into go-kev. db=sqlite3
INFO[08-19|19:47:31] Inserting VulnCheck Known Exploited Vulnerabilities... 
2827 / 2827 [---------------------------------------------------------------------------------------] 100.00% 3502 p/s
INFO[08-19|19:47:32] CveID Count                              count=2827

$ go-kev server

$ curl -s http://127.0.0.1:1328/cves/CVE-2023-43177 | jq
{
  "vulncheck": [
    {
      "vendorProject": "crushftp",
      "product": "CrushFTP",
      "shortDescription": "CrushFTP prior to 10.5.1 is vulnerable to Improperly Controlled Modification of Dynamically-Determined Object Attributes.",
      "vulnerabilityName": "crushftp CrushFTP Improper Control of Dynamically-Managed Code Resources",
      "required_action": "Apply remediations or mitigations per vendor instructions or discontinue use of the product if remediation or mitigations are unavailable.",
      "knownRansomwareCampaignUse": "Unknown",
      "cve": [
        {
          "cveID": "CVE-2023-43177"
        }
      ],
      "vulncheck_xdb": [
        {
          "xdb_id": "6d8fca1e85b0",
          "xdb_url": "https://vulncheck.com/xdb/6d8fca1e85b0",
          "date_added": "2023-12-27T00:06:20Z",
          "exploit_type": "initial-access",
          "clone_ssh_url": "git@github.com:the-emmons/CVE-2023-43177.git"
        }
      ],
      "vulncheck_reported_exploitation": [
        {
          "url": "https://go.recordedfuture.com/hubfs/reports/ta-2024-0321.pdf",
          "date_added": "2024-03-21T00:00:00Z"
        }
      ],
      "date_added": "2024-03-21T00:00:00Z"
    }
  ]
}
```

# Checklist:
You don't have to satisfy all of the following.

- [ ] Write tests
- [x] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [x] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES

# Reference

### CODE REVIEW

File: db/redis.go [hunk 1]
```diff
@@ -248,30 +266,144 @@ func (r *RedisDriver) InsertKEVulns(records []models.KEVuln) (err error) {
...
+	// newDeps, oldDeps: {"CVEID": {"VulnCheck:HashSum(CVEJSON)": {}}}
+	newDeps := map[string]map[string]struct{}{}
+	oldDepsStr := "{}"
+	t, err := r.conn.Type(ctx, depKey).Result()
+	if err != nil {
+		return xerrors.Errorf("Failed to Type key: %s. err: %w", depKey, err)
+	}
+	switch t {
+	case "string":
+		if _, err := r.conn.Del(ctx, depKey).Result(); err != nil {
```
Comment (shino) [inline db/redis.go:286]: In this case, should the value be inserted to KEV#DEP hash with CISA key after deletion?

Comment (MaineK00n) [reply db/redis.go:290]: fixed

https://github.com/vulsio/go-kev/compare/72ea3f9224d5d49c9cd8a6a5ddbcf2bff9343f31..f0e9def18b52f6d4133eb9325dadb9d6ecd93e8b

File: README.md [hunk 2]
```diff
@@ -43,6 +43,16 @@ INFO[11-16|04:39:00] Inserting Known Exploited Vulnerabilities...
 INFO[11-16|04:39:00] CveID Count                              count=291
 ```
 
+# Fetch VulnCheck Known Exploited Vulnerabilities
```
Comment (shino) [inline README.md:46]: How about mentioning https://docs.vulncheck.com/community/vulncheck-kev/attribution here?

**Review by shino — APPROVED** [review]
🎊

File: README.md [hunk 3]
```diff
@@ -43,6 +43,16 @@ INFO[11-16|04:39:00] Inserting Known Exploited Vulnerabilities...
...
+# Fetch VulnCheck Known Exploited Vulnerabilities (https://vulncheck.com/kev)
+```console
+$ go-kev fetch vulncheck
+INFO[08-23|02:34:55] Fetching VulnCheck Known Exploited Vulnerabilities 
+INFO[08-23|02:34:56] Insert VulnCheck Known Exploited Vulnerabilities into go-kev. db=sqlite3
+INFO[08-23|02:34:56] Inserting VulnCheck Known Exploited Vulnerabilities... 
+2832 / 2832 [------------------------------------------------------------------------------] 100.00% 2931 p/s
+INFO[08-23|02:34:57] CveID Count                              count=2832
+```
+
```
Comment (shino) [inline README.md:55]: How about inserting some sentence like this?

>  Before you use this data from VulnCheck, you MUST read https://docs.vulncheck.com/community/vulncheck-kev/attribution and make sure it's satisfied.

Comment (MaineK00n) [reply README.md:57]: https://github.com/vulsio/go-kev/compare/98dd60e38756c246e17a3bd2a8c86cbd35b6ca51..c904827b73654d59700a1980cd9c3f40a9395907

**Review by shino — APPROVED** [review]
🍻

**Review by shino — APPROVED** [review]
🎊

**Pull request merged**